### PR TITLE
NEPT-1097: Update the field.tpl, add a taxonomy term reference tpl,

### PR DESCRIPTION
**Pull request by Fefaine**
& remove legacy code.

### CODE REVIEW

**Comment by Fefaine** [comment]
@drupol strangely enough the preprocess function is not taken into account, @voidtek told me you would know why? Thanks !

**Comment by drupol** [comment]
A specific branch of Atomium needs to be used (nept-1093) to test this.

Please, let me know how it goes.

Thanks.

File: templates/field/field.tpl.php [hunk 1]
```diff
@@ -6,7 +6,7 @@
  */
 ?>
 <div<?php print render($attributes); ?>>
```
Comment (Fefaine) [inline templates/field/field.tpl.php]: why did you not correct this one? 😆

Comment (drupol) [inline templates/field/field.tpl.php]: Oops, I will fix this asap.

Comment (drupol) [inline templates/field/field.tpl.php]: There, I fixed it everywhere.

**Comment by Medi4tor** [comment]
It works. :+1:

File: templates/tags/tags.component.inc [hunk 2]
```diff
@@ -49,7 +56,35 @@ function ec_europa_atomium_definition_tags() {
...
+            '#tag' => 'span',
+            '#value' => $tag['value'],
+          );
+        }
+    }
+  }
+  else {
+    foreach ($variables['tags'] as $key => $tag) {
+      $variables['tags'][$key] = array(
+        '#theme' => 'link__' . $hook,
```
Comment (drupol) [inline templates/tags/tags.component.inc]: '#theme' => array('link__' . $variables['theme_hook_original']),

**Pull request merged**